src/ontology/Makefile

```diff
@@ -63,7 +63,7 @@ FORMATS_INCL_TSV = $(sort $(FORMATS) tsv)
 RELEASE_ARTEFACTS = $(sort $(ONT)-base $(ONT)-full )
 
 ifeq ($(ODK_DEBUG),yes)
-ODK_DEBUG_FILE = $(TMPDIR)/debug.log
```

Comment:
This and many other changes are coming from running `update_repo`?

Reply:
All changes to Makefile yes!